openapi.yaml

```diff
@@ -93,6 +93,12 @@ paths:
       operationId: getPublicKey
       tags:
         - pubkey
+      parameters:
+        - in: query
+          name: treeID
+          type: string
+          pattern: '^[0-9]+$'
+          description: The tree ID of the tree you wish to get a public key for
```

Comment:
Is this endpoint external for the user like `loginfo` or just internal?

Reply:
Is that documented anywhere? Maybe I missed it.

Reply:
Oh okay, thank you, I meant documented in Rekor for the user to be able to use some of these commands, but we can do that later if we want to. The link is helpful to me as I'm also trying to figure it out!